WordPress/src/main/java/org/wordpress/android/ui/reader/ReaderPostListFragment.java

```diff
@@ -450,6 +450,14 @@ public void onCreate(Bundle savedInstanceState) {
                     return null;
                 });
             });
+
```

Comment:
NitPick: Could we please remove the extra empty line :D.

Reply:
Right 😊. Removed here 0cdd939e4f1c9acff807ef2728f259f5ffcfc281